### ISSUE TITLE
Changing name of this repo?

### ISSUE THREAD

**Issue by holingpoon** [NYPL-discovery/sierraupdatepoller#33]
As per our conversation at our meeting, the code base is no longer only limited to polling for `UPDATE`s but also `DELETE`s happening on Sierra. Should the repo be renamed to `sierrapoller`?  Thanks!

**Comment by kfriedman** [comment]
Good point. I'm fine with this... I guess it's a little work to change git remotes and there might be some links out there pointing to this repo name. But the pros might outweigh the cons.
